__tests__/app.test.js

```diff
@@ -104,6 +104,68 @@ describe("GET /api/articles/:article_id", () => {
     expect(msg).toBe(expected);
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("Should return a 200 status", async () => {
+    const { status } = await request(app).get("/api/articles/1/comments");
+    expect(status).toBe(200);
+  });
+
+  test("Should return an array of comments requested by article id", async () => {
+    const {
+      body: { articleComments },
+    } = await request(app).get("/api/articles/1/comments");
+    const expected = {
+      comment_id: 9,
+      votes: 0,
+      created_at: "2020-01-01T03:08:00.000Z",
+      author: "icellusedkars",
+      body: "Superficially charming",
+      article_id: 1,
+    };
+
+    articleComments.forEach((comment) => {
+      expect(typeof comment.comment_id).toBe("number");
+      expect(typeof comment.votes).toBe("number");
+      expect(typeof comment.created_at).toBe("string");
+      expect(typeof comment.author).toBe("string");
+      expect(typeof comment.body).toBe("string");
+      expect(typeof comment.article_id).toBe("number");
```

Comment:
This is good, for a jest matcher that is tailor made for this type of thing check out `toMatchObject` - that will consolidate your many assertions into one